theia-rust-docker/bashrc

```diff
@@ -0,0 +1,19 @@
+#CONTAINER_START_PATH is passed as an env var when the container is created it is passed to the current user via the root environment (see sudoers)
+
+if [ -z "$CONTAINER_START_PATH" ]; then
+    echo "Using default start path"
+    export CONTAINER_START_PATH="/root"
+fi  
+
+if [ -z "$HOME" ]; then
+    export HOME="/root"
+fi
+
+echo "HOME: $HOME"
+echo "CONTAINER_START_PATH: $CONTAINER_START_PATH"
+cd "$CONTAINER_START_PATH"
```

Comment:
Is there an implicit assumption that CONTAINER_START_PATH be under the user's home directory, and so mounted in the container? Mine is not, and so when I execute `ex_bash`, I get at the end: 

```bash
CONTAINER_START_PATH: /opt/ericsson-dev/theia-apps/theia-rust-docker
bash: cd: /opt/ericsson-dev/theia-apps/theia-rust-docker: No such file or directory
```
If so it can be an acceptable limitation, but maybe add a note about it in `README.md`?

Reply:
Changed readme a bit. Cleaned up the ex command script. Will only change dir if it is possible.